### PR TITLE
Update to REL1_43 and 1.43.0

### PR DESCRIPTION
**Pull request by DanielWTQ**
== Version updates ==

Most extensions were tracking REL1_39 in gerrit and are now tracking REL1_43
and checking out the latest change as of 1.43.0 being tagged. Extensions that
were checking out a specific commit from the main branch that was not
associated with the release of a specific version of the extension were
switched to track REL1_43 if possible. For extensions and skins that use
dedicated releases, the following upgrades have been made:

* Pivot (skin) was changed from 2.3.0 to tracking REL1_43
* EmbedVideo: upgraded from 3.2.6 to 3.4.2
* ExternalData: upgraded from 3.2 to 3.3
* FlexDiagrams: upgraded from 0.5 to 0.6
* HeaderTabs: upgraded from 2.2 to 2.3
* Lingo: upgraded from a commit in the 3.2.2 development cycle to 3.2.4
* MintyDocs: upgraded from 1.0 to 1.2.1
* PageForms: upgraded from 5.6.1 to 5.9
* UserPageViewTracker: upgraded from 0.7 to the latest commit (unreleased)
* VEForAll: upgraded from 0.5.1 to 0.6
* WatchAnalytics: upgraded from 4.1.2 to 4.3
* PubmedParser: upgraded from 5.1.0 to 5.2.0
* TabberNeue: upgraded from a commit in the 1.7.5 development cycle to 2.7.1
* VariablesLua: upgraded from 1.5.0 to 1.6.0

For extensions not on gerrit, the latest commit to the main branch is generally
used.

== Available extensions ==

Some extensions are now bundled (and have been since MediaWiki 1.40) that were
previously manually cloned:
* DiscussionTools
* Echo
* Linter
* Thanks

The LoginNotify extension has also been bundled since 1.40; previously it was
not included in Taqasta.

The Renameuser extension, previously bundled with MediaWiki, was merged into
MediaWiki core in 1.40 and is no longer used as an extension.

Additionally, the Graph and SemanticDrilldown extensions were removed, and both
the AddMessages and Auth_remoteuser extensions have been added.

== Patch changes ==

* MediaWiki core no longer has a `sed` replacement to relax the
`monolog/monolog` dependency, core's monolog version changed in
wikimedia/mediawiki@09622ef2fc323f0f2fc3fb7e29993b36e8e7ad4a (and then again in
wikimedia/mediawiki@7015972c9d99b5b0de3e57e658912b5d72ade851) and the image
built fine for REL1_43 without trying to change the version used
* NCBITaxonomyLookup no longer has gerrit change 916452 applied, that was
merged as
wikimedia/mediawiki-extensions-NCBITaxonomyLookup@45253e71f8c95c05257daa6f69a56978e1909051
* GoogleLogin no longer cherry-picks gerrit change 1074530 directly, instead it
and change 1074514 are applied via the new `GoogleLogin-fixes.patch` patch file
* WSSlots has been switched to the WikiTeq fork
* CommentStreams checkout of
wikimedia/mediawiki-extensions-CommentStreams@567178f0eac7172536aac4aea20f4cd97b8ad891
has been removed, that commit is included in the REL1_43 branch
* PageForms patch for displaytitle fixes was rebased for PageForms 5.9
* PageForms cherry-pick of
wikimedia/mediawiki-extensions-PageForms@94ceca65c23a2894da1a26445077c786671aef0c
was removed, that commit is included in the 5.9 release
* Flow patch `flow-conversion-utils.patch` was removed, no longer needed
* GoogleAnalyticsMetrics has a new patch
`GoogleAnalyticsMetrics-relax-pin.patch` to relax a composer dependency version
pin

SEL-1925

### CODE REVIEW

**Comment by DanielWTQ** [comment]
Now at `REL1_43-20241211-af0ffea`

**Review by clarakosi — APPROVED** [review]
lgtm!

File: Dockerfile [hunk 1]
```diff
@@ -467,309 +452,304 @@ RUN set -x; \
...
 
 # E
 RUN set -x; \
 	cd $MW_HOME/extensions \
    	# EditAccount
 	&& git clone --single-branch -b $MW_VERSION https://gerrit.wikimedia.org/r/mediawiki/extensions/EditAccount $MW_HOME/extensions/EditAccount \
 	&& cd $MW_HOME/extensions/EditAccount \
-	&& git checkout -q a9a0d7002483d2165300afd108a8f4b3c2c485ff \
+	&& git checkout -q e59561c56836a98bc5f5f3068bc81fd67a5d1c41 \
 	# Flow
```
Comment (vedmaka) [inline Dockerfile:721]: We really should wipe the Flow from the image soon

**Review by vedmaka — DISMISSED** [review]

File: Dockerfile [hunk 2]
```diff
@@ -297,165 +293,154 @@ RUN set -x; \
...
 	# Description2
 	&& git clone --single-branch -b $MW_VERSION https://gerrit.wikimedia.org/r/mediawiki/extensions/Description2 $MW_HOME/extensions/Description2 \
 	&& cd $MW_HOME/extensions/Description2 \
-	&& git checkout -q d2a5322a44f940de873050573e35fba4eb3063f8 \
+	&& git checkout -q 50e2aef88053be12a66b617657c414a665e2d38e \
 	# Disambiguator
 	&& git clone --single-branch -b $MW_VERSION https://gerrit.wikimedia.org/r/mediawiki/extensions/Disambiguator $MW_HOME/extensions/Disambiguator \
 	&& cd $MW_HOME/extensions/Disambiguator \
-	&& git checkout -q b7e7fad5f9f3dccfb902a3cbfd3bf2b16df91871 \
+	&& git checkout -q 56a4738b35d331fe3d3f985b9e3d1c084ab75ed9	 \
```
[the remark this answers is not in the document]
Comment (DanielWTQ) [reply Dockerfile]: good catch, fixed that and some other whitespace issues

File: Dockerfile (hunk 1 above)
Comment (DanielWTQ) [reply Dockerfile:721]: yeah, but it'll probably be easier to migrate any uses on a 1.43 wiki than a 1.39 wiki

**Comment by DanielWTQ** [comment]
Self merging, only difference from approved patch is whitespace and typos in the commit message

**Pull request merged**